priv/labels/caltech101_labels.json

```diff
@@ -0,0 +1,104 @@
+{
+   "accordion": 0,
+   "airplanes": 1,
+   "anchor": 2,
+   "ant": 3,
+   "background_google": 4,
+   "barrel": 5,
+   "bass": 6,
+   "beaver": 7,
+   "binocular": 8,
+   "bonsai": 9,
+   "brain": 10,
+   "brontosaurus": 11,
+   "buddha": 12,
+   "butterfly": 13,
+   "camera": 14,
+   "cannon": 15,
+   "car_side": 16,
+   "ceiling_fan": 17,
+   "cellphone": 18,
+   "chair": 19,
+   "chandelier": 20,
+   "cougar_body": 21,
+   "cougar_face": 22,
+   "crab": 23,
+   "crayfish": 24,
+   "crocodile": 25,
+   "crocodile_head": 26,
+   "cup": 27,
+   "dalmatian": 28,
+   "dollar_bill": 29,
+   "dolphin": 30,
+   "dragonfly": 31,
+   "electric_guitar": 32,
+   "elephant": 33,
+   "emu": 34,
+   "euphonium": 35,
+   "ewer": 36,
+   "faces": 37,
+   "faces_easy": 38,
+   "ferry": 39,
+   "flamingo": 40,
+   "flamingo_head": 41,
+   "garfield": 42,
+   "gerenuk": 43,
+   "gramophone": 44,
+   "grand_piano": 45,
+   "hawksbill": 46,
+   "headphone": 47,
+   "hedgehog": 48,
+   "helicopter": 49,
+   "ibis": 50,
+   "inline_skate": 51,
+   "joshua_tree": 52,
+   "kangaroo": 53,
+   "ketch": 54,
+   "lamp": 55,
+   "laptop": 56,
+   "leopards": 57,
+   "llama": 58,
+   "lobster": 59,
+   "lotus": 60,
+   "mandolin": 61,
+   "mayfly": 62,
+   "menorah": 63,
+   "metronome": 64,
+   "minaret": 65,
+   "motorbikes": 66,
+   "nautilus": 67,
+   "octopus": 68,
+   "okapi": 69,
+   "pagoda": 70,
+   "panda": 71,
+   "pigeon": 72,
+   "pizza": 73,
+   "platypus": 74,
+   "pyramid": 75,
+   "revolver": 76,
+   "rhino": 77,
+   "rooster": 78,
+   "saxophone": 79,
+   "schooner": 80,
+   "scissors": 81,
+   "scorpion": 82,
+   "sea_horse": 83,
+   "snoopy": 84,
+   "soccer_ball": 85,
+   "stapler": 86,
+   "starfish": 87,
+   "stegosaurus": 88,
+   "stop_sign": 89,
+   "strawberry": 90,
+   "sunflower": 91,
+   "tick": 92,
+   "trilobite": 93,
+   "umbrella": 94,
+   "watch": 95,
+   "water_lilly": 96,
+   "wheelchair": 97,
+   "wild_cat": 98,
+   "windsor_chair": 99,
+   "wrench": 100,
+   "yin_yang": 101
+}
```

Comment:
Let's convert this to a map and embed it in Elixir directly. Less work to do when we actually get to download stuff :)